mj_envs/envs/env_base.py

```diff
@@ -110,6 +110,7 @@ def step(self, a):
         Step the simulation forward (t => t+1)
         Uses robot interface to safely step the forward respecting pos/ vel limits
         """
+        env_state = self.get_env_state()
```

Comment:
@suraj-nair-1  -- This seems like some kind of debug leftover. If not, can you please explain what is going on here and how you are using it?

Reply:
Yeah this was used during re-parsing state -> image demos, but is not needed anymore. Pushed a change to remove it